client/my-sites/pages/page/index.js

```diff
@@ -50,6 +50,8 @@ import isJetpackModuleActive from 'state/selectors/is-jetpack-module-active';
 import canCurrentUser from 'state/selectors/can-current-user';
 import config from 'config';
 
+import 'state/data-layer/wpcom/sites/homepage';
```

Comment:
Would it have the same effect if we preloaded when the the menu is opened? 

I tried it out in https://github.com/Automattic/wp-calypso/pull/45534

That way we're not affecting the bundle size.

Reply:
hey good call, I've updated it to load when you open the menu and that works 👍